config/scaffold/page/page.twig

```diff
@@ -0,0 +1,5 @@
+{% set layoutPath = '../../layouts/' ~ config.layout|capitalize ~ '/' ~ config.layout ~ '.twig' %}
```

Comment:
same problem with capitalize -> only makes the first letter Uppercase
but we need PascalCase here aswell